portal-kernel/src/com/liferay/portal/kernel/service/persistence/impl/BasePersistenceImpl.java

```diff
@@ -797,10 +822,21 @@ protected String getColumnName(
 		return fieldName;
 	}
 
+	protected FinderPath getDSLQueryFinderPath(
```

Comment:
Why does this method need to be implemented by sub classes?

Reply:
@Preston-Crary  Because I want its sub classes manage the service registration of FinderPath for dsl sql.